scripts/run_mypy_checks.py

```diff
@@ -751,6 +750,12 @@
     help='If true, skips installing dependencies. The default value is false.',
     action='store_true')
 
+_PARSER.add_argument(
+    '--ci-check',
```

Comment:
The flage name should somwhat match the `help` description, maybe `--install-globaly`?

Reply:
Done

Reply:
I agree with DubeySandeep's comment, but I don't see it fixed in the current diff

Reply:
Pushed it 1 minute before.